src/app/features/home/capture-tab/uploading-bar/uploading-bar.component.scss

```
@@ -4,15 +4,15 @@
   justify-content: stretch;
   transition: all 1s;
 
-  .upload-text {
+  .text-with-spacer {
     flex-grow: 0;
     font-size: 0.7em;
     margin-left: 10px;
     margin-right: 10px;
     color: darkgray;
   }
 
-  .complete-text {
+  .text-without-spacer {
     flex-grow: 1;
     font-size: 0.7em;
     margin-left: 10px;
```

```suggestion
.text {
  flex-grow: 1;
  font-size: 0.7em;
  margin-left: 10px;
  margin-right: 10px;
  color: darkgray;
}

.spacer-text {
  flex-grow: 0;
}
```

Maybe we can apply multiple CSS classes to reduce duplicated rules.